schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to generate alerts or other events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other events..
+   Examples of data sources that would populate the rule fields include: platforms, network or host IDS/IPS, network 
+   firewalls, web access firewalls, url filters, endpoint detection and response (EDR) systems, etc.  
+  
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Rule ID
+      description: >
+        A rule ID that is unique within the scope of an agent, observer, or other entity using the rule for detection of this event
+
+      example: 101 
+
+    - name: uuid
+      level: extended
+      type: keyword
+      short: Rule UUID
+      description: >
+        A rule ID that is unique within the scope of a set or group of agents, observers, or other entities using the rule for detection of this event.
+
+      example: 1100110011
+
+    - name: version
+      level: extended
+      type: keyword
+      short: Rule Version
+      description: The version / revision of the rule being used for analysis.
+      example: 1.1
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Rule Name
+      description: The name of the rule generating the event.
```

Comment:
I would add wording here about "name or signature". For example Suricata uses "signature" for the rule's name and "category" for categorization: https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/suricata/eve/test/eve-small.log-expected.json#L62-L67

Reply:
changed to "rule or signature"